farao-parent/farao-parent-ws/pom.xml

```diff
@@ -54,13 +55,22 @@
                 <version>${springfox.version}</version>
             </dependency>
 
+            <!-- Import dependencies -->
             <dependency>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-dependencies</artifactId>
                 <version>${springboot.dependencies.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+
+            <!-- Test dependencies -->
+            <dependency>
+                <groupId>org.junit.platform</groupId>
+                <artifactId>junit-platform-launcher</artifactId>
+                <version>${junit.launcher.version}</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
We want to reduce at most the number of dependencies defined here and we discuss everytime we want to add a new one. It looks like this is used to handle a very specific problem for some developers that can be can fixed other ways so this won't be added to farao-parent.

Reply:
removed. To add in child-repo instead, when needed